README.md

```diff
@@ -113,7 +113,12 @@ An implementation is not compliant if it fails to satisfy one or more of the MUS
 
 # Provisioned Service
 
-A Provisioned Service resource **MUST** define a `.status.binding` field which is a `LocalObjectReference`-able (containing a single field `name`) to a `Secret`.  The `Secret` **MUST** exist in the same namespace as the resource.  The `Secret` data **SHOULD** contain a `type` entry with a value that identifies the abstract classification of the binding.  The `Secret` type (`.type` verses `.data.type` fields) **SHOULD** reflect this value as `servicebinding.io/{type}`, substituting `{type}` with the `Secret` data type.  It is **RECOMMENDED** that the `Secret` data also contain a `provider` entry with a value that identifies the provider of the binding.  The `Secret` data **MAY** contain any other entry.  To facilitate discoverability, it is **RECOMMENDED** that a `CustomResourceDefinition` exposing a Provisioned Service add `servicebinding.io/provisioned-service: "true"` as a label.
+A Provisioned Service resource **MUST** create a Secret resource which is referenced in on of these ways:
+
+1. a `.status.binding` field which is a `LocalObjectReference`-able (containing a single field `name`) to a `Secret`.
+2. `.metadata.name` pointing to the Secret
+
+The `Secret` **MUST** exist in the same namespace as the resource.  The `Secret` data **SHOULD** contain a `type` entry with a value that identifies the abstract classification of the binding.  The `Secret` type (`.type` verses `.data.type` fields) **SHOULD** reflect this value as `servicebinding.io/{type}`, substituting `{type}` with the `Secret` data type.  It is **RECOMMENDED** that the `Secret` data also contain a `provider` entry with a value that identifies the provider of the binding.  The `Secret` data **MAY** contain any other entry.  To facilitate discoverability, it is **RECOMMENDED** that a `CustomResourceDefinition` exposing a Provisioned Service add `servicebinding.io/provisioned-service: "true"` as a label.
 
```

Comment:
@baijum one other possibility I can think of is checking first for a `servicebinding.io/binding-fieldpath` annotation on a CR and if found, use that information to find the field within the Spec or Status that houses the Binding. That way, things like ACK could store the binding in, say, `Status.ACKResourceMetadata.ServiceBinding` and set an annotation on CR's of `servicebinding.io/binding-fieldpath=status.ackResourceMetadata.serviceBinding`. This would be extensible and allow for use cases where the CRD's schema is generated or cannot otherwise accomodate a `Status.Binding` field.

Reply:
@jaypipes Using annotation to specify the binding field path is a good idea. If the annotation is missing in a Provisioned Service, the default location would be the current field path (i.e., `.status.binding`). So, this change would be backward compatible.